### PR TITLE
Print disk usage

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

File: .azure/pipelines/ci.yml [hunk 1]
```diff
@@ -332,6 +332,8 @@ stages:
             arguments: $(Build.SourcesDirectory)/NuGet.config $Token
           env:
             Token: $(dn-bot-dnceng-artifact-feeds-rw)
+      - script: df -h
+        displayName: Disk utilization (before)
```
Comment (dougbu) [inline .azure/pipelines/ci.yml:336]: Suggest doing this in &hellip;/jobs/default-build.yml instead with new steps excluding Windows i.e. preceded by
```yml
- ${{ if ne(parameters.agentOs, 'Windows') }}:
```

And, as @MattGal suggested, let's do this for realz.

File: .azure/pipelines/ci.yml [hunk 2]
```diff
@@ -653,6 +659,8 @@ stages:
           testResultsFormat: 'xUnit'
           testResultsFiles: '*.xml'
           searchFolder: '$(Build.SourcesDirectory)/artifacts/TestResults/$(_BuildConfig)/Quarantined'
+      - script: df -h
+        displayName: Disk utilization (before)
```
Comment (Pilchie) [inline .azure/pipelines/ci.yml:663]: This is after.

**Review by Pilchie — APPROVED** [review]

File: .azure/pipelines/ci.yml (hunk 1 above)
Comment (pranavkm) [reply .azure/pipelines/ci.yml]: Updatd

File: .azure/pipelines/jobs/default-build.yml [hunk 3]
```diff
@@ -193,6 +196,10 @@ jobs:
 
     - ${{ parameters.afterBuild }}
 
+    - ${{ if ne(parameters.agentOs, 'Windows') }}:
+      - script: df -h && du -h --threshold=50MB /home/vsts/work/
```
Comment (dougbu) [inline .azure/pipelines/jobs/default-build.yml:200]: Unfortunately macOS doesn't support `--threshold` (oops). Suggest using `-d 4` or similar on that `agentOS`.

File: .azure/pipelines/jobs/default-build.yml [hunk 4]
```diff
@@ -193,6 +196,10 @@ jobs:
 
     - ${{ parameters.afterBuild }}
 
+    - ${{ if ne(parameters.agentOs, 'Windows') }}:
```
Comment (dougbu) [inline .azure/pipelines/jobs/default-build.yml:199]:
```suggestion
    - ${{ if eq(parameters.agentOs, 'Linux') }}:
```

File: .azure/pipelines/jobs/default-build.yml (hunk 3 above)
Comment (dougbu) [inline .azure/pipelines/jobs/default-build.yml:200]:
```suggestion
      - script: df -h && du -h --threshold=50MB /home/vsts/work/
    - ${{ if eq(parameters.agentOs, 'macOS') }}:
      - script: df -h && du -h -d 4 ..
```

File: .azure/pipelines/jobs/default-build.yml [hunk 5]
```diff
@@ -193,6 +196,12 @@ jobs:
 
     - ${{ parameters.afterBuild }}
 
+    - ${{ if eq(parameters.agentOs, 'Linux') }}:
+      - script: df -h && du -h --threshold=50MB /home/vsts/work/
```
Comment (dougbu) [inline .azure/pipelines/jobs/default-build.yml:200]:
```suggestion
      - script: df -h && du -h --threshold=50MB /home/vsts/work/
        displayName: Disk utilization
```

File: .azure/pipelines/jobs/default-build.yml [hunk 6]
```diff
@@ -193,6 +196,13 @@ jobs:
...
```
Comment (dougbu) [inline .azure/pipelines/jobs/default-build.yml:200]:
```suggestion
      - script: df -h && du -h --threshold=50MB ..
```

File: .azure/pipelines/jobs/default-build.yml [hunk 7]
```diff
@@ -193,6 +196,13 @@ jobs:
 
     - ${{ parameters.afterBuild }}
 
+    - ${{ if eq(parameters.agentOs, 'Linux') }}:
+      - script: df -h && du -h --threshold=50MB ..
+        displayName: Disk utilization
+    - ${{ if eq(parameters.agentOs, 'macOS') }}:
+      - script: df -h && du -h -d 4 ..
```
Comment (dougbu) [inline .azure/pipelines/jobs/default-build.yml:203]:
```suggestion
      - script: df -h && du -h -d 3 ..
```

**Pull request merged**